src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -346,7 +346,7 @@ public partial class ArrayTypeMismatchException : System.SystemException
         public ArrayTypeMismatchException(string message, System.Exception innerException) { }
     }
     public delegate void AsyncCallback(System.IAsyncResult ar);
-    [System.AttributeUsageAttribute((System.AttributeTargets)(32767), Inherited=true, AllowMultiple=false)]
+    [System.AttributeUsageAttribute(System.AttributeTargets.All, Inherited=true, AllowMultiple=false)]
```

Comment:
It's great to see these values fixed to be readable :)

Reply:
Yep, definitely!